openmc/data/neutron.py

```diff
@@ -633,6 +633,18 @@ def from_ace(cls, ace_or_filename, metastable_scheme='nndc'):
             rx = Reaction.from_ace(ace, i)
             data.reactions[rx.mt] = rx
 
+        # If present, use fission xs to compute "fission heating" coefficient
+        fission_reaction = data.reactions.get(18)
+        if fission_reaction is not None:
+            fission_xs = fission_reaction.xs[strT].y
+
+            # Compute "fission-less" heating coefficient
+            no_fission_heating = Reaction(999)
+            no_fission_heating.xs[strT] = Tabulated1D(
+                energy, heating_number * (total_xs - fission_xs))
```

Comment:
This assumes that the heating number from fission and other reactions is the same, which is not true. What we need to be doing is taking the total heating (MT=301) and subtract the fission heating as given by NJOY (MT=318). Part of the challenge of this is that the fission heating is not present in the ACE file, so we'd only have it available when we're running `from_njoy` in which case we can pull it out of the PENDF (pointwise ENDF) file. If you look at the `IncidentNeutron.from_njoy` method, you can see an example of pulling data out of the PENDF file (0K elastic scattering cross section which is used for resonance upscattering methods).

That being said, NJOY is just using the MT=458 data to calculate kermas, so I think we should be able to back out what MT=318 should have been by multiple the fission xs by the appropriate sum of components of energy release.